### PR TITLE
viewallproduct and homepage

### PR DESCRIPTION
**Pull request by buitannguyen2612**
view of this page
 - HomePage
    image
    image
    image
-ViewAllProduct
    image
    image
    image
    image
    image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,9 +13,12 @@
     "@emotion/styled": "^11.11.0",
     "@mui/icons-material": "^5.14.11",
     "@mui/material": "^5.14.11",
+    "axios": "^1.5.1",
+    "keen-slider": "^6.8.6",
     "next": "latest",
     "react": "latest",
-    "react-dom": "latest"
+    "react-dom": "latest",
+    "react-paginate": "^8.2.0"
```
Comment (thdailong) [inline package.json:21]: Isn't we already have MUI pagination?

File: app/ViewAllProduct/AllProduct/pages/until.js
Comment (thdailong) [inline app/ViewAllProduct/AllProduct/pages/until.js:1]: What is until.js??

**Review by thdailong — CHANGES_REQUESTED** [review]
Please format your code before commiting!
You can use prettier

File: package.json (hunk 1 above)
Comment (buitannguyen2612) [reply package.json:21]: fixed

File: app/ViewAllProduct/AllProduct/pages/until.js
Comment (buitannguyen2612) [reply app/ViewAllProduct/AllProduct/pages/until.js:1]: fixed

**Review by thdailong — APPROVED** [review]

**Pull request merged**